### PR TITLE
Add meeting notes for 2020-01-15

### PR DESCRIPTION
**Pull request by jkrems**
Closes https://github.com/nodejs/modules/issues/466

### CODE REVIEW

File: doc/meetings/2020-01-15.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+* **Minutes Google Doc**: https://docs.google.com/document/d/1odxHwuFIuZsbaQ9SgH23riGiwvKg0y2pHlTwdaIOIbE/edit
+
+## Present
+
+* Jan Krems (@jkrems)
+* Jordan Harband (@ljharb)
+* Myles Borins (@MylesBorins)
+* Guy Bedford (@guybedford) _GB_
+* Geoffrey Booth (@GeoffreyBooth) _GFB_
+* Wesley Wigham (@weswigham)
```
Comment (GeoffreyBooth) [inline doc/meetings/2020-01-15.md]: Plus @evanplaice, others?

Comment (jkrems) [reply doc/meetings/2020-01-15.md]: Copied from the doc, @coreyfarrell was also present I think..? If people were present but forgot to add themselves in the doc, feel free to use the suggestions feature on this PR to add yourselves after the fact. :)

Comment (coreyfarrell) [inline doc/meetings/2020-01-15.md]:
```suggestion
* Wesley Wigham (@weswigham)
* Corey Farrell (@coreyfarrell)
* Evan Plaice (@evanplaice)
```

Sorry I never made it into the Google doc, hopefully this suggested change works correctly (I thought there was an issue with multi-line suggestions)

Comment (jkrems) [reply doc/meetings/2020-01-15.md]: Looks like it worked! Thanks. :)

File: doc/meetings/2020-01-15.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
+# Node.js Foundation Modules Team Meeting 2020-01-15
+
+## Links
+
+* **Recording**:
```
Comment (DerekNonGeneric) [inline doc/meetings/2020-01-15.md]: https://www.youtube.com/watch?v=SuFC_yzc51A

Comment (jkrems) [reply doc/meetings/2020-01-15.md]:
```suggestion
- * **Recording**:
+ * **Recording**: https://www.youtube.com/watch?v=SuFC_yzc51A
```

**Pull request merged**